packages/starship/build.sh

```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://starship.rs
 TERMUX_PKG_DESCRIPTION="A minimal, blazing fast, and extremely customizable prompt for any shell"
 TERMUX_PKG_LICENSE="ISC"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.48.0
+TERMUX_PKG_VERSION=0.50.0
 TERMUX_PKG_SRCURL=https://github.com/starship/starship/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=23e729ace48ec0bf6d8eff5f99003351463841f3b28fe453faceb62e6f99bae6
+TERMUX_PKG_SHA256=d8f4dc9bd266f2a5c34926d361c62fdddb61cd7da4acadba5f9c175eb07602e5
 TERMUX_PKG_DEPENDS="zlib"
 TERMUX_PKG_BUILD_IN_SRC=true
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--no-default-features"
```

Comment:
```suggestion
TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--locked --no-default-features --features http"
```
`--locked` is required as a temporary workaround for https://github.com/termux/termux-packages/pull/6432#issuecomment-795384017

Reply:
@vladimyr Thanks for the help! What does `--features http` do here?

Reply:
HTTP library is used for Starship's bug reporting CLI command :upside_down_face: